qtism/data/storage/xml/Utils.php

```diff
@@ -205,7 +205,7 @@ public static function getDOMElementAttributeAs(DOMElement $element, $attribute,
 
         switch ($datatype) {
             case 'string':
-                return $attr;
+                return htmlspecialchars_decode($attr);
```

Comment:
Same as here https://github.com/oat-sa/qti-sdk/pull/324/files#r952473967

Reply:
Fixed by the new approach where we are not double encoding the attributes as it is implicitly performed by PHP through DOMElement.